index.js

```diff
@@ -0,0 +1,59 @@
+
+function calculate (input) {
+    //numberfy stringy numbers
+    for (let i=0; i < input.length; i++) {
+        const element = input[i];      
+        //check for strings
+        if ((typeof(element)) === 'string') {
+            //check for numbers in those strings
+            if (!isNaN(element)) {
+                // change stringy numbers to regular numbers
+                input[i] = parseInt(element);
+            }
+        }
```

Comment:
This can be condensed to just a single if-statement by using the '&&' operator to combine the two conditions being checked against.